frontend/pages/admin/OrgSettingsPage/cards/Advanced/Advanced.tsx

```diff
@@ -194,6 +229,38 @@ const Advanced = ({
               error={formErrors.host_expiry_window}
             />
           )}
+          <Checkbox
+            onChange={handleInputChange}
+            name="deleteActivities"
+            value={deleteActivities}
+            parseTarget
+            tooltipContent={
+              <>
+                When enabled, allows automatic cleanup of audit logs older than
+                the number of days specified in the{" "}
+                <em>Audit log retention window</em>
+                setting.
+                <em>
+                  (Default: <strong>Off</strong>)
+                </em>
+              </>
+            }
+          >
+            Delete activities
+          </Checkbox>
+          {deleteActivities && (
+            <Dropdown
+              searchable={false}
+              options={activityExpiryWindowOptions}
+              onChange={onChangeSelectActivityExpiryWindow}
+              placeholder="Select"
+              value={activityExpiryWindow}
+              label="Max activity age"
+              name="Max activity age"
+              parseTarget
+              error={formErrors.activity_expiry_window}
```

Comment:
Are you sure error state works right like this? the `name` field might have to line up with the formErrors field name.

Reply:
Actually, since we're not setting any errors for this field (makes sense, user can only choose from the valid values we provide or their already-validated custom value), no need to set the error state here at all.

Reply:
Ahh yeah, fixed!